README.md

```diff
@@ -136,22 +133,25 @@ def hp_tuning_select_best_model(
   <img height=500 src="assets/03_train.png">
 </p>
 
-Having the best model architecture and its hyperparameters defined in the previous stage makes it possible to train a quality model. Also, model training is the right place to bring an [Experiment Tracker](https://docs.zenml.io/stacks-and-components/component-guide/experiment-trackers) into the picture - we will log all metrics and model itself into the [Experiment Tracker](https://docs.zenml.io/stacks-and-components/component-guide/experiment-trackers), so we can register our model in a [Model Registry](https://docs.zenml.io/stacks-and-components/component-guide/model-registries) and pass it down to a [Model Deployer](https://docs.zenml.io/stacks-and-components/component-guide/model-deployers) easily and traceable. We will use information from Active Stack to make implementation agnostic of the underlying infrastructure.
+Having the best model architecture and its hyperparameters defined in the previous stage makes it possible to train a quality model object. Also, model training is the right place to bring an [Experiment Tracker](https://docs.zenml.io/stacks-and-components/component-guide/experiment-trackers) into the picture - we will log all metrics and model object itself into the [Experiment Tracker](https://docs.zenml.io/stacks-and-components/component-guide/experiment-trackers), so we can register our model object in a [Model Registry](https://docs.zenml.io/stacks-and-components/component-guide/model-registries) and pass it down to a [Model Deployer](https://docs.zenml.io/stacks-and-components/component-guide/model-deployers) easily and traceable. We will use information from Active Stack to make implementation agnostic of the underlying infrastructure.
+To make the most out of the Model Control Plane we additionally annotate output model object as a Model Artifact, by doing so it will be properly categorized for future use and get additional model object specific features.
 <details>
   <summary>Code snippet 💻</summary>
 
 ```python
+from zenml.model import ModelArtifactConfig
+
 experiment_tracker = Client().active_stack.experiment_tracker
 @step(experiment_tracker=experiment_tracker.name)
 def model_trainer(
     ...
-) -> Annotated[ClassifierMixin, "model"]:
+) -> Annotated[ClassifierMixin, "model", ModelArtifactConfig()]:
   ...
 ```
 </details>
-Even knowing that the hyperparameter tuning step happened we would like to ensure that our model meets at least minimal quality standards - this quality gate is on the evaluation step. In case the model is of low-quality metric-wise an Exception will be raised and the pipeline will stop.
+Even knowing that the hyperparameter tuning step happened we would like to ensure that our model object meets at least minimal quality standards - this quality gate is on the evaluation step. In case the model object is of low-quality metric-wise an Exception will be raised and the pipeline will stop.
```

Comment:
```suggestion
Even knowing that the hyperparameter tuning step happened we would like to ensure that our model object meets at least minimal quality standardsy this quality gate is on the evaluation step. In case the model object is of low quality metric-wise an `Exception` will be raised and the pipeline will stop.
```